### PR TITLE
Add snapshot repository for Jellyfin SDK

### PR DESCRIPTION
**Pull request by Maxr1998**

### CODE REVIEW

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 1]
```diff
@@ -88,7 +92,8 @@ object Dependencies {
     }
 
     object Network {
-        const val jellyfinSdk = "org.jellyfin.sdk:jellyfin-platform-android:${Versions.jellyfinSdk}"
+        const val jellyfinSdk = "${Groups.jellyfin}:jellyfin-platform-android:${Versions.jellyfinSdk}"
+        const val jellyfinSdkSnapshot = "${Groups.jellyfin}:jellyfin-platform-android:latest-SNAPSHOT"
```
[the remark this answers is not in the document]
Comment (Maxr1998) [reply buildSrc/src/main/kotlin/Dependencies.kt:96]: lol, I had that at first. I could add that back, but I'll soon migrate to version catalogs anyway. So, skip?

File: build.gradle.kts [hunk 2]
```diff
@@ -17,6 +17,12 @@ allprojects {
     repositories {
         mavenCentral()
         google()
+        maven {
+            setUrl("https://s01.oss.sonatype.org/content/repositories/snapshots/")
```
[the remark this answers is not in the document]
Comment (Maxr1998) [reply build.gradle.kts:21]: Oh, good shorthand. Interesting though that it's not marked as inline.

File: app/build.gradle.kts [hunk 3]
```diff
@@ -126,7 +127,14 @@ dependencies {
...
 
     // Network
-    implementation(Dependencies.Network.jellyfinSdk)
+    val useSdkSnapshot: String by project
+    implementation(Dependencies.Network.jellyfinSdk) {
+        version {
+            if (parseBoolean("useSdkSnapshot", useSdkSnapshot)) {
+                strictly(Dependencies.Versions.jellyfinSdkSnapshot)
+            }
+        }
```
[the remark this answers is not in the document]
Comment (Maxr1998) [reply app/build.gradle.kts:136]: That doesn't work? :eyes:
`version` is a string..

[the remark this answers is not in the document]
Comment (nielsvanvelzen) [reply app/build.gradle.kts:136]: oh

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**